CI fails with simplesat 0.9.0

The CI stage `sv-tests.ci` fails when installing dependencies.
Offending dependency is `simplesat==0.9.0`.

Failure can be observed here: https://github.com/chipsalliance/sv-tests/pull/6031/checks

```
17:13:18 | Collecting simplesat>=0.8.0
17:13:18 |   Downloading simplesat-0.9.0-py3-none-any.whl (248 kB)
17:13:18 |      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 248.8/248.8 KB 27.1 MB/s eta 0:00:00
17:13:18 | ERROR: Error decoding metadata for /tmp/pip-unpack-mmw7reno/simplesat-0.9.0-py3-none-any.whl: 'utf-8' codec can't decode byte 0x82 in position 1851: invalid start byte in METADATA file
Error: Process completed with exit code 1.
```

The last working CI is from 2 days ago (before release of simplesat 0.9.0 and it did run with `simplesat==0.8.2`.

--- a/conf/requirements.txt
+++ b/conf/requirements.txt
@@ -9,6 +9,7 @@ make_var
 markupsafe
 pytablewriter
 GitPython
+simplesat==0.8.2
 git+https://github.com/lowRISC/edalize.git@ot
 git+https://github.com/lowRISC/fusesoc.git@ot
 orderedmultidict

